ko/news/_posts/2020-09-25-ruby-3-0-0-preview1-released.md

```diff
@@ -40,19 +40,20 @@ module ChatApp
 end
```

Comment:
translated~ d917191e89de52d14000cd3a525825b9c

Reply:
👍 `prime?` code below also can be translated.

Reply:
updated 43f4a035
btw, I think what `expr1, expr2` means is quite ambiguous 🤔
It may be better that to translate whole sentence to `r1, r2의 실행 결과 대기`..

Reply:
That sounds good to me.

Reply:
fixed 9d84b9c4